project-koku/koku
# Masu endpoint to re-download ingress reports

## Jira Ticket

[COST-3908](https://issues.redhat.com/browse/COST-3908)

## Description

This change will add a new masu endpoint so we can view and re-download failed ingress reports from customer filtered flows

## Testing

1. Checkout Branch
2. Restart Koku
3. Add a storage only source
4. Post some data using the /api/cost-management/v1/ingress/reports/ api
5. Use the masu endpoint to check posted reports
6. use the masu endpoint to kick off a fresh download.


## Notes

...
